src/components/layout/playground/requests/requestCard.tsx

```diff
@@ -132,17 +137,24 @@ const PlaygroundRequestCard: React.FC<
           >
             {requester.name}
           </Li>
-          <Li
-            title={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'} ${
-              timeUntilDue || ''
-            }
+          {/*If the dueDate strings are null, return a custom "Due Date: None" <Li>, else insert a <Li> which says when the due date is */}
+          {timeUntilDue === null ? (
+            <Li title={'Due Date'} name={'Due Date'} category={category}>
+              None
+            </Li>
+          ) : (
+            <Li
+              title={`${
+                timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'
+              } ${timeUntilDue || ''}
             ${timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}`}
-            name={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
-            category={category}
-          >
-            {timeUntilDue}
-            {timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}
-          </Li>
+              name={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
+              category={category}
+            >
+              {timeUntilDue}
+              {timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}
+            </Li>
+          )}
```

Comment:
As commented before you can't check here for null.
Please also try to just adapt the given structure for the new value. Something like this:
`            title={`${
              timeUntilDue
                ? hasNoDue
                  ? 'None'
                  : isDue
                  ? 'Was due'
                  : 'Due in'
                : 'Due'
            }`
Regarding the comment here, after scaling it down like this it shouldn't be necessary as well, but for comments like this try not to explain what is happing, but why it is there.
In this case something like 'Handle requests without dueDate' - bad example since it is self-explanatory.